image/postgres/rhtap.Dockerfile

```
@@ -0,0 +1,60 @@
+FROM registry.access.redhat.com/ubi8/ubi-minimal:latest as builder
+
+COPY . /stackrox
+
+FROM registry.redhat.io/rhel8/postgresql-13 as final
```

Let's make it explicit that we use the `latest` tag.

```suggestion
FROM registry.redhat.io/rhel8/postgresql-13:latest AS final
```